neighbour-model/src/test/resources/application.properties

```diff
@@ -1,6 +1,10 @@
-spring.datasource.url=jdbc:postgresql://localhost:7080/federation
+#
+# In memory DB for testing repository
+#
+spring.datasource.url=jdbc:h2:~/testdb
```

Comment:
Please try testing against the docker component database.
We introduced that because embedded h2-database did not break failed O/R-mapping.